docs/userguide.md

```diff
@@ -56,9 +56,22 @@ Kubernetes environments that have been tested and are supported by the Federatio
 
 After completing the steps in one of the above guides, return here to continue the Federation v2 deployment.
 
+### Set Cluster Environment Variables
+
+Please set the following variables in order to continue through this guide.
+Replace the name of the clusters with the respective name of the clusters
+you're using if they differ.
+
+```bash
+HOST_CLUSTER=cluster1
```

Comment:
(No action required) I have mixed feelings about adding this indirection.  It's likely fine from a programmers' perspective, but I think less friendly for a user.   It might have made more sense to name the clusters `hostcluster` and `membercluster` instead of adding environment variables to the mix.